### PR TITLE
Fix compilation error due to header file for Mutex missing

### PR DESCRIPTION
**Pull request by JeevakaPrabu**
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>

### CODE REVIEW

**Comment by JeevakaPrabu** [comment]
Compilation error seen in Android O + brillo.

**Comment by JeevakaPrabu** [comment]
@kalyankondapally

**Comment by JeevakaPrabu** [comment]
@pyn1

File: os/android/hwcservice.h [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 #include <binder/IInterface.h>
 #include <utils/String8.h>
+#include <utils/Mutex.h>
```
Comment (kalyankondapally) [inline os/android/hwcservice.h]: Lets get rid of mutex usage and use spinlock like rest of code base.

**Comment by kalyankondapally** [comment]
merged

**Pull request closed without merging**